pilot/pkg/xds/ecds.go

```diff
@@ -48,6 +52,8 @@ func ecdsNeedsPush(req *model.PushRequest) bool {
 			return true
 		case gvk.WasmPlugin:
 			return true
+		case gvk.Secret:
```

Comment:
comment on L48 is stale. But more importantly I am seeing Secret pushes filtered out entirely at a higher level (ProxyNeedsPush). Because of https://github.com/istio/istio/blob/8ce0defcc905873dadf3fa1d8c3f3629cd39895c/pilot/pkg/xds/proxy_dependencies.go#L26

Reply:
Fixed. I was only testing with Gateway and did not find this :).